pimpmycause/profiles/migrations/0002_auto_20160730_2059.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.8 on 2016-07-30 20:59
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('profiles', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='pimpuser',
+            name='linkedin',
+            field=models.CharField(default='http://twitter.com', max_length=30),
```

Comment:
Migrations cover how to apply changes to the database after the model was modified. So if i were to add the default url and then remove it there would be 2 migrations: 1 to show how to add the default and then one more to remove it. It is basically a step by step to apply to the database based in changes. 

So I had the defaults set and then realised I don't need to so I removed them. Hence why you can see this in a migration. 

Custom user: django default users only store email, password and username. If we want to store more details about the user we must extend or override the default django user. Since we are not authenticacing the user by username + password (pimp wireframes use email and password) we need to amend the django user to authenticate with email + password. So the changes in this PR are to completely override the default django user with the help of a third party app that makes email the required field needed for a user to login. Then on top of that I added all the required Pimp fields like name, surname that is not available by default.